code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -907,7 +909,19 @@
 		for(var/obj/item/alien_embryo/embryo in potential_host)
 			embryo.hivenumber = XENO_HIVE_FORSAKEN
 		potential_host.update_med_icon()
+	var/shipside_humans_count = 0
+	var/xenos_count = 0
+	for(var/mob/living/carbon/human/current_human as anything in GLOB.alive_human_list)
+		if(isspecieshuman(current_human) || isspeciessynth(current_human))
+			var/turf/turf = get_turf(current_human)
+			if(is_mainship_level(turf?.z))
+				shipside_humans_count++
+	for(var/mob/living/carbon/xenomorph/xenomorph as anything in totalXenos)
```

Comment:
Get the count of xenos when the list is iterated on line 888; also only count xenos that are hijacking (that loop is already figuring out who isn't - e.g. just `continue` after `stored_larva++` and then tabbed one left add your tally `xeno_count++`).

Reply:
Done and I think I fixed fugger issue, although probably it can be coded better. Please take a look.

Reply:
Be sure to add this to your changelog as a fix

Reply:
Oh wait forgot another check

Reply:
Now it should work

Reply:
Yeah that makes sense 👍